languages/CookieConsent/se.ini

```diff
@@ -0,0 +1,32 @@
+Accept All Cookies = "Dohkket buot niesttážiid" 
```

Comment:
This file should be in the upstream vufind version. You can add these with Lokalise or with a pull request to vufind-org/vufind.

Reply:
Or I can create the pull request on your behalf if you wish so.

Reply:
I added this now, Lokalise seems to be working very nicely